app/controllers/subscription_controller.rb

```diff
@@ -0,0 +1,18 @@
+class SubscriptionController < ApplicationController
+	def index
+    end
+    def create
+        begin
+        @list_id = 'fe525f1cb2'
```

Comment:
let's put this also in .env file as i think we want to create testing list on staging